flocker/acceptance/testtools.py

```diff
@@ -157,6 +151,10 @@ def _clean_node(test_case, node):
     # http://doc-dev.clusterhq.com/advanced/cleanup.html#removing-zfs-volumes
     # A tool or flocker-deploy option to purge the state of a node does
     # not yet exist. See https://clusterhq.atlassian.net/browse/FLOC-682
+    containers = _run_SSH(22, 'root', node, ['docker', 'ps', '-aq'], None)
+    containers = containers.split(b"\n")[:-1]
+    for c in containers:
```

Comment:
One letter variable names are bad for readability and maintainability (in my opinion), please change this to `container`.